swig/cpp/examples/process_tree.cpp

```diff
@@ -75,6 +75,13 @@ int main() {
         auto schema_list = std::shared_ptr<std::vector<S_Schema_Node>>(node->schema()->tree_dfs());
         for(auto elem = schema_list->begin() ; elem != schema_list->end() ; ++elem) {
             std::cout << "schema name " << (*elem)->name() << " type " << (*elem)->nodetype() << std::endl;
+            if  (LYS_LEAF == (*elem)->nodetype()) {
```

Comment:
This is not related to Doxygen, and therefore it should be made in a separate commit.